cli/exchange/business.go

```diff
@@ -111,70 +112,80 @@ func BusinessAddPolicy(org string, credToUse string, policy string, jsonFilePath
 }
 
 //BusinessUpdatePolicy will replace a single attribute of a business policy in the Horizon Exchange
-func BusinessUpdatePolicy(org string, credToUse string, policy string, attribute string, valueFilePath string) {
+func BusinessUpdatePolicy(org string, credToUse string, policyName string, attribute string) {
```

Comment:
Is it better to use file to pass in the patch? -f <file name> or -f- for stdin. Please refer to "hzn exchange service publish` command for how to specify a file. Some patches are very long like "userInput".